uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Test/Fabricator/SicknessRequest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
+use CRM_HRLeaveAndAbsences_BAO_SicknessRequest as SicknessRequest;
+use CRM_HRLeaveAndAbsences_Test_Fabricator_LeaveBalanceChange as LeaveBalanceChangeFabricator;
+
+class CRM_HRLeaveAndAbsences_Test_Fabricator_SicknessRequest {
+
+  private static $leaveRequestStatuses;
+  private static $dayTypes;
+  private static $sicknessReasons;
+
+  public static function fabricate($params, $withBalanceChanges = false) {
```

Comment:
why not  $withBalanceChanges just be a parameter ( you can unset it inside the method ) to keep consistency  ?